src/style.css

```diff
@@ -1,3 +1,63 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
+  box-sizing: border-box;
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font-family for Roboto in case it fails to load. For example, 'Roboto, Arial, sans-serif'.